dds_registration/models.py

```diff
@@ -86,14 +88,22 @@ def __init__(self, *args, **kwargs):
 
 
 class Membership(models.Model):
-    user = models.ForeignKey(User, related_name='registrations', on_delete=models.CASCADE)
-    started = models.IntegerField(default=lambda: date.today().year)
-    until = models.IntegerField(default=lambda: date.today().year)
+    user = models.ForeignKey(User, related_name='memberships', on_delete=models.CASCADE)
+    # NOTE: Using different date comparsion method, using real dates, not only years...
+    started = models.IntegerField(default=this_year)
+    #  # OLD CODE: It causes an error during migration creation: `ValueError: Cannot serialize function: lambda`
+    #  started = models.IntegerField(default=lambda: date.today().year)
+    #  until = models.IntegerField(default=lambda: date.today().year)
     honorary = models.BooleanField(default=False)
 
+    @property
+    def until(self):
```

Comment:
Please revert this, one could have membership from e.g. 2020 to 2022 but they aren't active now. `until` needs to be a column.